### ISSUE TITLE
Create a Page Demonstrating How To Build a Form with Legos

### ISSUE THREAD

**Issue by jonathanprozzi** [HausDAO/sdk-docs#43]

**Comment by jonathanprozzi** [comment]
@earth2travis Working on this at the same time as the Form Builder docs

**Comment by jonathanprozzi** [comment]
Duplicate of another issue. I accidentally made it here I guess. This'll be closed out once the pow. pr is merged in. @earth2travis
